sys/dev/e1000/if_em.c

```diff
@@ -37,7 +37,8 @@
 /*********************************************************************
  *  Driver version:
  *********************************************************************/
-char em_driver_version[] = "7.6.1-k";
+char em_driver_version[] = "7.7.8-fbsd";
+char igb_driver_version[] = "2.5.19-fbsd";
```

Comment:
Incrementing these to avoid confusion in users; we are roughly on par with these OOT versions.